x-pack/platform/plugins/shared/stack_connectors/common/openai/schema.ts

```diff
@@ -34,6 +34,31 @@ export const ConfigSchema = schema.oneOf([
     defaultModel: schema.string(),
     headers: schema.maybe(schema.recordOf(schema.string(), schema.string())),
   }),
+  schema.object({
+    apiProvider: schema.oneOf([schema.literal(OpenAiProviderType.PkiOpenAi)]),
+    apiUrl: schema.string(),
+    defaultModel: schema.string({ defaultValue: DEFAULT_OPENAI_MODEL }),
+    certificateFile: schema.maybe(
+      schema.oneOf([
+        schema.string({ minLength: 1 }),
+        schema.arrayOf(schema.string({ minLength: 1 }), { minSize: 1 }),
+      ])
+    ),
+    certificateData: schema.maybe(schema.string({ minLength: 1 })),
+    privateKeyFile: schema.maybe(
+      schema.oneOf([
+        schema.string({ minLength: 1 }),
+        schema.arrayOf(schema.string({ minLength: 1 }), { minSize: 1 }),
+      ])
+    ),
+    privateKeyData: schema.maybe(schema.string({ minLength: 1 })),
+    verificationMode: schema.oneOf([
+      schema.literal('full'),
+      schema.literal('certificate'),
+      schema.literal('none'),
+    ], { defaultValue: 'full' }),
```

Comment:
I'm wondering if this should default to `none` since there is currently no verification mode implemented?